app/controllers/v0/caregivers_assistance_claims_controller.rb

```diff
@@ -39,19 +39,23 @@ def download_pdf
       # Brakeman will raise a warning if we use a claim's method or attribute in the source file name.
       # Use an arbitrary uuid for the source file name and don't use the return value of claim#to_pdf
       # as the source_file_path (to prevent changes in the the filename creating a vunerability in the future).
-      source_file_path = PdfFill::Filler.fill_form(@claim, SecureRandom.uuid, sign: false)
+      source_file_path = if Flipper.enabled?(:caregiver1010)
+                           @claim.to_pdf(SecureRandom.uuid,
+                                         sign: false)
+                         else
+                           PdfFill::Filler.fill_form(
+                             @claim, SecureRandom.uuid, sign: false
+                           )
+                         end
+
       client_file_name = file_name_for_pdf(@claim.veteran_data)
       file_contents    = File.read(source_file_path)
 
-      File.delete(source_file_path) if !Flipper.enabled?(:caregiver1010) && File.exist?(source_file_path)
-
       auditor.record(:pdf_download)
 
       send_data file_contents, filename: client_file_name, type: 'application/pdf', disposition: 'attachment'
     ensure
-      if Flipper.enabled?(:caregiver1010) && (source_file_path && File.exist?(source_file_path))
-        File.delete(source_file_path)
-      end
+      File.delete(source_file_path) if source_file_path && File.exist?(source_file_path)
```

Comment:
Deleting usage of the toggle for this. It has been validated in staging and is currently turned on in production. I could move this to it's own PR if it's an issue, but I thought since this change was small it would be fine.